cves/CVE-2013-0863.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Buffer overflow in the rle_decode functin in libavcode/sanm.c in FFmpg before 1.0.4
+and 1.1.x before 1.1.2 allows attackers to impact system via crafted LucasArtsSmush video data. This
+could lead to system crashes or cause a denial of service condition
```

Comment:
I'd explain what `LucasArtsSmush` is. If it's LucasArts, I'd bet people would be interested, so make sure to explain the format and what it's used for, for both understanding and some flair.